framework/src/main/java/org/checkerframework/framework/util/typeinference8/types/InferenceFactory.java

```diff
@@ -1032,7 +1032,10 @@ public ConstraintSet getCheckedExceptionConstraints(
         for (UseOfVariable ei : es) {
           constraintSet.add(
               new Typing(
-                  new ProperType(iter2.next(), xi, context), ei, TypeConstraint.Kind.SUBTYPE));
+                  String.format("Exception Constraint for  %s", expression),
```

Comment:
Minor: in some cases, String.format can be replaced by `+` which is more efficient.